imap_processing/cdf/config/imap_ultra_l1b_variable_attrs.yaml

```
@@ -0,0 +1,310 @@
+default_attrs: &default
+  # Assumed values for all variable attrs unless overwritten
+  DEPEND_0: epoch
+  DISPLAY_TYPE: time_series
+  FILLVAL: -9223372036854775808
+  FORMAT: I12
+  VALIDMIN: -9223372036854775808
+  VALIDMAX: 9223372036854775807
+  VAR_TYPE: data
+
+epoch:
+  <<: *default
+  CATDESC: Time, number of nanoseconds since J2000 with leap seconds included
+  FIELDNAM: epoch
+  LABLAXIS: epoch
+  UNITS: ns
+  VAR_TYPE: support_data
+  SCALETYP: linear
+  MONOTON: INCREASE
+  TIME_BASE: J2000
+  TIME_SCALE: Terrestrial Time
+  REFERENCE_POSITION: Rotating Earth Geoid
+
+x_front:
+  <<: *default
+  CATDESC: x front position
+  FIELDNAM: Event x-position on front foil
+  LABLAXIS: x front position
+  # TODO: come back to format
+  UNITS: mm
+
+y_front:
+  <<: *default
+  CATDESC: y front position
+  FIELDNAM: Event y-position on front foil
+  LABLAXIS: y front position
+  # TODO: come back to format
+  UNITS: mm
+
+x_back:
+  <<: *default
+  CATDESC: x back position
+  FIELDNAM: x_back
+  LABLAXIS: x back position
+  # TODO: come back to format
+  UNITS: mm
+
+y_back:
+  <<: *default
+  CATDESC: y back position
+  FIELDNAM: y_back
+  LABLAXIS: y back position
+  # TODO: come back to format
+  UNITS: mm
+
+x_coin:
+  <<: *default
+  CATDESC: x coincidence position
+  FIELDNAM: x_coin
+  LABLAXIS: x coincidence position
+  # TODO: come back to format
+  UNITS: mm
+
+tof_start_stop:
+  <<: *default
+  CATDESC: Particle time of flight from start to stop
+  FIELDNAM: tof_start_stop
+  LABLAXIS: tof start stop
+  # TODO: come back to format
+  UNITS: ns
+
+tof_stop_coin:
+  <<: *default
+  CATDESC: Particle time of flight from stop to coincidence
+  FIELDNAM: tof_stop_coin
+  LABLAXIS: tof stop coin
+  # TODO: come back to format
+  UNITS: ns
+
+tof_corrected:
+  <<: *default
+  CATDESC: Corrected time of flight
+  FIELDNAM: tof_corrected
+  LABLAXIS: tof corrected
+  # TODO: come back to format
+  UNITS: ns
+
+eventtype:
+  <<: *default
+  CATDESC: Eventtype (1-2 top and bottom stop types; 8-15 ssd stop types)
+  FIELDNAM: eventtype
+  LABLAXIS: eventtype
+  # TODO: come back to format
+  UNITS: ' '
```

Can this be an empty string, or does it have to have a space in it?
```suggestion
  UNITS: ''
```

---

Bryan Harter says it needs a space.

---

Bummer, another CDF annoyance for me to live with ;) Thanks for clarifying!